File: app/src/main/java/de/tum/in/tumcampusapp/models/gcm/GCMAlert.java
package de.tum.in.tumcampusapp.models.gcm;

import java.io.Serializable;

/**
 * Used for parsing the GCM json payload for the 'alert' type
 */
public class GCMAlert implements Serializable {

    private static final long serialVersionUID = 3906290674499996501L;
    public boolean silent;

}

File: app/src/main/java/de/tum/in/tumcampusapp/models/gcm/GCMAlert.kt
package de.tum.`in`.tumcampusapp.models.gcm

import java.io.Serializable

/**
 * Used for parsing the GCM json payload for the 'alert' type
 */
data class GCMAlert(var silent: Boolean) : Serializable {
    companion object {
        private const val serialVersionUID = 3906290674499996501L
    }
}
